src/constructors.ts

```diff
@@ -1,4 +1,5 @@
 import * as Yup from 'yup'
+import { ValidateOptions } from 'yup/lib/types'
```

Comment:
This import seems a bit brittle, but this type doesn't seem to be exported from `yup`.